requirements.txt

```diff
@@ -1,3 +1,14 @@
-docker==4.0.*
-mypy==0.720
+pydocstyle==3.*
```

Comment:
The last pydocstyle version is 4.0.0 Why do we need 3.* version?

Reply:
Cause have error with last version.
https://github.com/sphericalpm/lektorium/pull/37

Reply:
Ok, then i think it will be better to fixed the last version obviously.

Reply:
Fix it in my requirements.